### DISCUSSION TITLE
Rule to enforce single quote strings

### DISCUSSION THREAD

**Discussion by Orlando-pt** [stoplightio/spectral#2601] [Rulesets]
Is there any rule to enforce single quote strings? Or can we add any?

**Comment by uuf6429** [comment]
I'm guessing you mean for yaml, right? 'Coz json requires double-quotes by design.

I've a feeling that it wouldn't be possible since spectral checks the content, not the source syntax (e.g. it won't be able to check things like spaces, indentation etc), but I may be wrong... I only started using this tool today.
